### DISCUSSION TITLE
Como aplicar o destroy sem apagar o bucket usado no Backend?

### DISCUSSION THREAD

**Discussion by LuigNascimento** [chgasparoto/curso-aws-com-terraform#50] [Geral]
Depois de criarmos o bucket para o backend s3, como fazemos para apagar qualquer outra coisa criada no momento do apply, exceto o bucket do backend?

**Comment by chgasparoto** [comment]
@LuigNascimento você precisa criar esse bucket separado do restante da sua infra, dessa forma ele vai ter um **tfstate** específico para ele e independente das alterações feitas em outras pastas não irá afetá-lo.

**Reply by fabioffalcao** [reply]
Professor,

Mas qual a vantagem de criar apenas esse único bucket do tfstate via terraform? Teria algum problema criar apenas esse primeiro bucket do tfstate via console?

**Reply by chgasparoto** [reply]
@fabioffalcao não teria nenhum problema criar o bucket na mão. 
Mas caso você tenha mais de uma conta na AWS e mais de um ambiente para dar manutenção você pode tirar vantagem do Terraform para gerenciá-los.

**Reply by fabioffalcao** [reply]
Show de bola, nesse caso seria interessante apenas manter o state desses buckets separados mesmo né? Valeu

**Reply by chgasparoto** [reply]
Isso 👍
